tests/ansible/ocsp_admin.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Do we need to store these certs and CSRs? To my understanding they will be generated by the test.

Reply:
Yes, you are right, this was a wrong commit. Removed all certs.